scripts/setup-adapters.sh

```diff
@@ -29,6 +29,8 @@ if [[ "$OSTYPE" == darwin* ]]; then
   export INSTALL_PREFIX=${INSTALL_PREFIX:-"$(pwd)/deps-install"}
 fi
 
+WGET_OPTIONS=${WGET_OPTIONS:-"-q"}
```

Comment:
should be empty by default.

Reply:
WGET_OPTIONS=${WGET_OPTIONS:-""}

Reply:
I am not sure, as per @czentgr in the internal PR the default behaviour was to silent the logs. So I am following the same here.